src/geometry/GeoTrackView.i.hh

```diff
@@ -174,12 +188,15 @@ CELER_FUNCTION real_type GeoTrackView::move_by(real_type dist)
 //! Update state to next volume
 CELER_FUNCTION void GeoTrackView::relocate()
 {
+#ifdef VECGEOM_USE_NAVINDEX
     if (vgnext_.Top() != nullptr)
     {
-        Vec3D tmpPos(detail::to_vector(this->pos_));
+        // BVH requires an lvalue temp_pos
+        auto temp_pos = detail::to_vector(this->pos_);
         Navigator::RelocateToNextVolume(
```

Comment:
```suggestion
        detail::BVHNavigator::RelocateToNextVolume(
```

Reply:
Hmm rebase error thanks. I'll check on wildsdtyle and emmet if the ci isn't working